include/dev/RedundantADC.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <EVT/io/ADC.hpp>
+
+namespace IO = EVT::core::IO;
+
+namespace RedundantADC {
+
+/**
+ * This class allows processing readings from multiple ADCs and checking for redundancy.
+ */
+class RedundantADC {
+public:
+    /**
+     * @brief Enum class representing the status of RedundantADC processing.
+     */
+    enum class Status {
+        OK = 0,                /**< No error. */
+        OFF_BY_ONE_ERROR = 1,  /**< The reading is off by 1*/
+        MARGIN_ERROR = 2,      /**< The reading is off by less than the specific margin*/
+        COMPARISON_ERROR = 3,  /**< The readings do not match.*/
+        CAN_LOOPBACK_ERROR = 4 /**< CAN loopback data does not match.*/
+    };
+
+    /**
+     * @brief Constructs a RedundantADC object with the given ADC instances.
```

Comment:
We don't use `@brief`
```suggestion
     * Construct a RedundantADC object with the given ADC instances.
```